build.sbt

```diff
@@ -390,16 +1012,22 @@ lazy val mimaSettings = Seq(
 def ignoreUndocumentedPackages(packages: Seq[Seq[java.io.File]]): Seq[Seq[java.io.File]] = {
   packages
     .map(_.filterNot(_.getName.contains("$")))
-    .map(_.filterNot(_.getCanonicalPath.contains("io/delta/sql")))
+    .map(_.filterNot(_.getCanonicalPath.contains("kernel")))
+    .map(_.filterNot(_.getCanonicalPath.contains("connectors")))
```

Comment:
Made unidoc ignore all non-Spark related source code files for now. Even though we are adding these unidoc settings only to the spark module, unidoc tends to process the source files of all the modules. Unless we explicitly ignore them, unidoc generation seems to fail in all sorts of odd ways.

In a future PR, i would like to consider adding flink and standard docs to this unidoc